bc_obps/reporting/models/rls_configs/activity_json_schema.py

```diff
@@ -0,0 +1,14 @@
+from reporting.enums.enums import ReportingTableNames
+from rls.enums import RlsRoles, RlsOperations
+from rls.utils.helpers import generate_rls_grants
+
+
+class Rls:
+    role_grants_mapping = {
+        RlsRoles.INDUSTRY_USER: [RlsOperations.SELECT],
+        RlsRoles.CAS_DIRECTOR: [RlsOperations.SELECT],
+        RlsRoles.CAS_ADMIN: [RlsOperations.SELECT],
+        RlsRoles.CAS_ANALYST: [RlsOperations.SELECT],
+        RlsRoles.CAS_VIEW_ONLY: [RlsOperations.SELECT],
+    }
+    grants = generate_rls_grants(role_grants_mapping, ReportingTableNames.ACTIVITY_JSON_SCHEMA)
```

Comment:
As a note for some potential refactor: That call could be generic for every model, since the table name can be derived from the meta class of the model. The grant mappings should be sufficient as configuration